File: pe/visualstudio/PE.FSharp/Problem0007.fs
namespace PE.FSharp.Problem7

open Microsoft.FSharp.Core 
open Microsoft.FSharp.Core.Operators 
open FSharp.MyLibraries.Primes


module NthPrime =

    let nthPrime nth  = 
        List.nth (listOfPrimes (nth * 100)) (nth - 1) 

File: pe/visualstudio/PE.CSharp/Problem0007.cs
using System.Linq;
using FSharp.MyLibraries;

namespace PE.CSharp
{
    public class Problem0007
    {
        public int NthPrime(int nth)
        {
            return Primes
                .listOfPrimes(nth * 100)
                .ElementAt(nth - 1);
        }
    }
}
